react-components/src/communities/view/CommunityListing.js

```diff
@@ -60,40 +60,44 @@ class CommunityListing extends Component {
         } = this.props;
 
         return (
-            <div className={classes.wrapper}>
+            <Fragment>
                 {loading &&
                 <CircularProgress size={30} classes={{root: classes.loading}} thickness={7}/>
                 }
-                <Table>
-                    <EnhancedTableHead selectable={false}
-                                       rowCount={data ? data.length : 0}
-                                       order={order}
-                                       orderBy={orderBy}
-                                       baseId={parentId}
-                                       ids={ids.TABLE_HEADER}
-                                       columnData={tableColumns}
-                                       onRequestSort={this.onRequestSort}
-                    />
-                    <TableBody>
-                        {(!data || data.length === 0) && <EmptyTable message={getMessage("noCommunities")} numColumns={tableColumns.length}/>}
-                        {data && data.length > 0 && stableSort(data, getSorting(order, orderBy)).map(community => {
-                            return (
-                                <TableRow tabIndex={-1}
-                                          hover
-                                          key={community.id}>
-                                    <TableCell>
-                                        <DEHyperLink onClick={() => onCommunityClicked(community)}
-                                                     text={collaboratorsUtil.getSubjectDisplayName(community)}/>
-                                    </TableCell>
-                                    <TableCell>
-                                        {community.description}
-                                    </TableCell>
-                                </TableRow>
-                            )
-                        })}
-                    </TableBody>
-                </Table>
-            </div>
+                <div className={classes.table}>
```

Comment:
Does adding `className={classes.table}` to the `Table` component work? Or is this `div` wrapper required?

Reply:
Oh, good point - I'll double check

Reply:
Seems the div wrapper is required. Otherwise, if you only have 1 row in the table, it becomes a really tall row to fill up the rest of the space